### PR TITLE
pkg/cli: Use rowStrIter in call to printQueryOutput

### PR DESCRIPTION
**Pull request by richardartoul**
Fixes conflict caused by #15524

### CODE REVIEW

File: pkg/cli/cert.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func runListCerts(cmd *cobra.Command, args []string) error {
 			fmt.Sprintf("user=%s", name)})
 	}
 
-	printQueryOutput(os.Stdout, certTableHeaders, rows, "", cliCtx.tableDisplayFormat)
+	printQueryOutput(os.Stdout, certTableHeaders, newRowSliceIter(rows), "", cliCtx.tableDisplayFormat)
```
Comment (tamird) [inline pkg/cli/cert.go]: I think you need to return this now. Also, could you rebase? Another PR was just merged.

**Comment by tamird** [comment]
Could you include "Closes #15523." In your commit message when you address the failure? Thanks for fixing this!

**Comment by richardartoul** [comment]
Woops, good catch. Made the change quickly and forgot the interface changed to return an error.

**Comment by richardartoul** [comment]
Rebased and made the change

**Comment by tamird** [comment]
LGTM will merge on green. Thanks @richardartoul! :heart:

**Comment by richardartoul** [comment]
No problem, thanks for letting me contribute. I'll try and find another task to work on.

**Pull request merged**